vyper/parser/external_call.py

```diff
@@ -76,10 +76,8 @@ def external_contract_call(node,
     ]
     if context.is_constant() and not sig.const:
         raise ConstancyViolationException(
-            "May not call non-constant function '%s' within %s." % (
-                method_name,
-                context.pp_constancy(),
-            ) +
+            f"May not call non-constant function '{method_name}' within {context.pp_constancy()}."
+            +
```

Comment:
After converting lines 79-82 to an f-string literal the `+` in this line is no longer needed - python will implicitly convert to a single literal.